packages/contracts-bedrock/src/L2/SoulGasToken.sol

```diff
@@ -17,7 +17,7 @@ contract SoulGasToken is ERC20Upgradeable, OwnableUpgradeable {
     struct SoulGasTokenStorage {
         // _minters are be whitelist EOAs, only used when !IS_BACKED_BY_NATIVE
         mapping(address => bool) _minters;
-        // _burners are whitelist EOAs, only used when !IS_BACKED_BY_NATIVE
```

Comment:
OP's [coding style](https://github.com/ethstorage/optimism/blob/develop/packages/contracts-bedrock/STYLE_GUIDE.md) suggest that "Immutable variables  should be internal and have a hand written getter function"

Reply:
I don't get it here, `_burners` is not immutable.

Reply:
I was referring `IS_BACKED_BY_NATIVE`

Reply:
Ah, good suggestion, done.